xc/xenops_server_xen.ml

```diff
@@ -3507,6 +3507,25 @@ let look_for_xenctrl () =
       exit 1;
     end
 
+let watch_xen_event () =
+  let open Xeneventchn in
+  let handle = init () in
+  let virq_enomem_port = bind_virq handle Enomem in
+  let host_uuid = Inventory.lookup Inventory._installation_uuid in
+  let rec process_event () =
+    debug "waiting for xen event at (%s)" __LOC__;
+    let port = pending handle in
+    if port = virq_enomem_port then begin
+      info "Sending HOST_LOW_MEMORY message.";
+      let pid = Forkhelpers.safe_close_and_exec None None None [] !Xc_resources.send_message_script ["--name"; "HOST_LOW_MEMORY"; "--priority"; "2"; "--class"; "host"; "--uuid"; host_uuid; "--body"; ""]  in
```

Comment:
Let's fill in the body as well. A short text to explain what this is about.

Reply:
OK

Reply:
I'm not seeing it ;)

Reply:
You mean the message `body`, I misunderstand to code body.🤣

Reply:
Yes, I meant the `"--body"; ""` :) I see it now.